kinetic/distribution.yaml

```diff
@@ -0,0 +1,14 @@
+%YAML 1.1
+# ROS distribution file
+# see REP 143: http://ros.org/reps/rep-0143.html
+---
+release_platforms:
+  fedora:
+  - '22'
+  - '23'
```

Comment:
Then we will likely not support ROS on Fedora 25 in any ROS distros for the majority of it's lifetime. It is currently scheduled to drop 6 months before the release of ROS-L (assuming patterns hold).

I can't argue with your logic, but I want to make sure we all understand what the repercussions of leaving f25 off from the list are.